allspice/apiobject.py

```diff
@@ -1835,6 +1835,21 @@ def commit(self):
         }
         self._commit(args)
 
+    def download(self) -> bytes:
```

Comment:
For large files, this will use a lot of memory. Is this really the API we want to provide?

Reply:
It's admittedly expanding scope, but I imagine something like `release.download_to_file(path)` that uses [this](https://stackoverflow.com/questions/16694907/download-large-file-in-python-with-requests) to stream the bytes to a file.

Reply:
Added.